services/wallet/router.go

```diff
@@ -694,22 +529,22 @@ func (r *Router) suggestedRoutes(
 						continue
 					}
 
-					if !sendType.isAvailableBetween(network, dest) {
```

Comment:
This check is implemented in `AvailableFor` function of each type individually.

Reply:
After second thought:
```
func (s SendType) isAvailableBetween(from, to *params.Network) bool {
	if s.IsCollectiblesTransfer() {
		return from.ChainID == to.ChainID
	}

	if s == Bridge {
		return from.ChainID != to.ChainID
	}

	if s == Swap {
		return from.ChainID == to.ChainID
	}

	return true
}
```

This was use to prevent path to be created if send type prevent it. e.g u want to do a bridge only then you should only allow from and dest  that are different only. Are u still checking that, I cannot see it?

Reply:
Yes, all conditions you see here are actually covered and they are the first check in corresponding `AvailableFor` function implementations for each type that implements `Bridge` interface.

For example in case of Hop bridge that implementation looks like this:
```
func (h *HopBridge) AvailableFor(from, to *params.Network, token *token.Token, toToken *token.Token) (bool, error) {
	if from.ChainID == to.ChainID {
		return false, nil
	}
	.
	.
	.
	return true, nil
}
```

In case of Celar bridge looks like this:
```
func (s *CBridge) AvailableFor(from, to *params.Network, token *token.Token, toToken *token.Token) (bool, error) {
	if from.ChainID == to.ChainID {
		return false, nil
	}
	.
	.
	.
	return true, nil
}
```